chunky/src/java/se/llbit/chunky/ui/SceneChooserController.java

```diff
@@ -173,7 +178,11 @@ public void setStage(Stage stage) {
   private void populateSceneTable(File sceneDir) {
     List<SceneListItem> scenes = new ArrayList<>();
     List<File> fileList = SceneHelper.getAvailableSceneFiles(sceneDir);
-    Collections.sort(fileList);
+    // TODO: Sort by recently modified instead of file name.
+    Collections.sort(fileList, Comparator
+            .comparing((File o) -> -(Long)o.lastModified()) // negative of last modified -> most recent at top
+            .thenComparing((File o) -> o.toString().toLowerCase()) // Then by file name, ignoring case.
```

Comment:
```suggestion
            .thenComparing(Comparator.naturalOrder()) // Then by File::compareTo the pathnames lexicographically
```

Reply:
Lexicographical order sorts all uppercase characters prior to all lower case characters, and that can be confusing if someone is looking for something by a certain name, and it happens to have different capitalization and thus not show up where they wanted, which is why I specifically chose to sort by lowercase strings.

But I suppose that is a little less important now that it is being sorted by last modified time... I would prefer to keep it this way, unless LeMaik wants otherwise.

Reply:
Good call, I haven't checked if my simpler code results in the same sorting. I'll check it right now!

Reply:
Uppercase before lowercase is indeed not ideal. On the other hand, names with suffix numbers are ordered somewhat more coherent (which is not that important because a matching last modified timestamp happens rather infrequently).